glossary.yml

```diff
@@ -244,9 +244,14 @@
 
     It's useful for [debugging - TODO]() too.
 
+- term: Power user
```

Comment:
Power user is not about having "powers".


> A power user is a user of computers, software and other electronic devices, who uses advanced features of computer hardware, operating systems, programs, or websites which are not used by the average user. A power user might not have extensive technical knowledge of the systems they use but is rather characterized by competence or desire to make the most intensive use of computer programs or systems.
>
> https://en.wikipedia.org/wiki/Power_user

Reply:
Mmmh, I went too much "litteral" here (happy frenglish... x) ). What about:

> Codewars' users gain honor in different ways. Doing so, they progressively unlock some [privileges](references/gamification/privileges) giving them access to extra features like the possibility to approve kata, maintain kata they didn't author, resolve comments, ...

?

Reply:
It's not about gaining privileges. If they don't actively make use of them, they're not power users.

I think the key point in the Wikipedia is:

> **desire** to make the most intensive use

---

The opposite of power user is average user. Those users that only uses Codewars to train are average users even if they have many privileges unlocked. They don't use features like authoring kata, or get involved with maintaining Codewars.

Users' activities can be broken down like the following:

1. Train
2. Review
3. Create/Translate
4. Maintain

Probably 99% of the users only does 1 and partially 2 (solving beta). 3 and 4 are power users that affects Codewars.